### DISCUSSION TITLE
add voice cloning

### DISCUSSION THREAD

**Discussion by Manni1000** [ThioJoe/Auto-Synced-Translated-Dubs#50] [Ideas]
would be cool.
i think they have a api and they write some stuff about different languages on there web page you probably have to ask but the voices sound extremely good. https://beta.elevenlabs.io/
the demo has 10.000 free char of voice cloning

**Comment by ThioJoe** [comment]
Gonna convert this to a discussion

**Reply by Manni1000** [reply]
have you tried the voice cloning on the web page?

**Reply by ThioJoe** [reply]
I haven't tried the voice cloning. The TTS sounds really good but it's 16x more expensive than Azure so it wouldn't be cost effective for me to use.

**Reply by Manni1000** [reply]
i hope that someone will make something like this open source so we can use it.

**Reply by Pythone** [reply]
> i hope that someone will make something like this open source so we can use it.

I also hope it would be easy to run locally on almost any modern computer(like TranslateLocally/Firefox Translations)

**Comment by ThioJoe** [comment]
The latest version of the script now supports Eleven Labs API

**Reply by mbunga23** [reply]
Friend, I'm Miguel and I followed all the steps given in the repository, but I understand little about programming, I would like to expand my channel to different languages ​​by converting the src files into audio. If you have a video on how to do everything step by step, it would help a lot brother.

**Comment by mbunga23** [comment]
Friend, I'm Miguel and I followed all the steps given in the repository, but I understand little about programming, I would like to expand my channel to different languages ​​by converting the src files into audio. If you have a video on how to do everything step by step, it would help a lot brother.
